components/Cell.js

```diff
@@ -125,20 +128,20 @@ const Cell = props => {
       case 'Detail':
         return (
           <View style={_styles.cellAccessoryView}>
-            <View style={_styles.accessory_detail}>
+            <TouchableOpacity style={_styles.accessory_detail} onPress={onPressDetail} activeOpacity={0.7} disabled={isDisabled}>
```

Comment:
Since the cell is using TouchableHighlight, use of TouchableOpacity on accessory detail prevents the cell highlighting being triggered. I believe the look and feel is native. You can find some detail disclosure within iOS settings, they use opacity as well. :)